.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,36 @@
+name: Bug report
+description: Report a bug.
+labels: ['type:bug', 'status:requirements', 'priority-5-triage']
```

Comment:
Note that we will need to do #59 first before we can merge this PR, or the labels won't apply properly. 😄